guides/rosey-migration-guide/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+_schema: guide-item
+_uuid: 62b7d0c6-28d6-4375-a8dd-1c0d12f60f39
+_created_at: '2023-05-26 00:32:03 +0000'
+title: Introduction
+nav_title: Introduction
+order: 1
+published: true
+image: /documentation/static/CloudCannonDocumentationog.jpg
+description: >-
+  Learn about Rosey i18n and how to use it in CloudCannon
+tags: []
+related_articles:
+related_links:
+guide_listing:
+  show_on_index: true
+  category: CloudCannon Guides
+---
+If you’re already using CloudCannon’s i18n feature, you might want to consider moving to Rosey. Rosey is an open-source tool developed by CloudCannon, which does the same job a bit better. In this guide, we’ll first discuss some of the advantages of using Rosey over CloudCannon’s built-in i18n functionality. Then, we’ll look at a step-by-step walkthough you can use to migrate your site.
+
+## Advantages of Rosey
+
+- Rosey is open-source, so if there’s something you don’t like, you can create an issue or GitHub or even submit a pull request to have it changed. The code is completely public if you want to check out what’s happening under the hood.
+- Rosey is written in Rust, and runs blazingly fast.
+- Since Rosey is available as an NPM package, you can develop and test your translated site locally, instead of pushing to CloudCannon to build the site every time you make a change.
+- Rosey has a nicer file format than CloudCannon’s i18n, but is also completely backwards-compatible.
+- Rosey has a much wider feature set
+    - With the `--wrap` flag, you can ensure that languages with no spaces are appropriately wordwrapped for better readability and accessibility.
+    - Tooling to easily check which parts of the site haven’t been translated, or where the translations have become outdated from the original text.
+    - Support for translating images, JSON, and HTML attributes like img alt tags and meta descriptions.
+
+## Caveats
+
+One problem with using Rosey with CloudCannon is that it can interfere with the Visual Editor. The Visual Editor needs to know where your site's output files are, and in the workflow described in this guide Rosey moves those files around. For this reason, we recommend using Rosey **only** in conjunction with a staging workflow.
```

Comment:
```suggestion
One consideration when integrating Rosey with CloudCannon is that it can interfere with the Visual Editor. The Visual Editor needs to know where your site's output files are, and in the workflow described in this guide Rosey moves those files around. For this reason, we recommend using Rosey **only** in conjunction with a staging workflow.
```